### ISSUE TITLE
Alternative to BountySource?

### ISSUE THREAD

**Issue by hmans** [crystal-lang/crystal#6051]
First of all, I apologize for creating an issue for a non-technical topic -- to me, it felt like the cleanest way of raising this question/issue with the core team in a manner that makes it publicly visible and findable. If this was a stupid idea, please close this and accept my apologies.

My question is the following: I would like to support Crystal with a monthly donation. **Is there an alternative way of doing this besides going through BountySource?**

My reason for asking this question is that my experience with BountySource so far has not been great. A short summary:

- Signing up via GitHub failed with some error message I unfortunately did not record
- Signing up via Twitter asked me for my email address, then gave me an error that my email was already in use (possibly an artifact of my previous attempt at signing up through GitHub)
- I eventually gained access to my account through BountySource's **password recovery**
- Setting up a $10/month donation for Crystal gave me a **meaningless error message** ("Validation failed: Active support levels count Can't modify active rewards"), but only _after_ authorizing with PayPal
- Two more attempts at setting up a $10/month subscription had the same result
- The next day, **I see a $30 charge from BountySource**. Lo and behold, they did, in fact, set up three separate $10/month subscriptions.
- The UI for cancelling these subscriptions is broken (the submit button doesn't close the modal), but fortunately, the actual cancellation works.

I know these are bugs with BountySource and not Crystal (and I've raised them with BountySource in my [slightly angry rant on Twitter](https://twitter.com/hmans/status/991324634695061504)), but at this point, I simply don't trust BountySource enough for _any_ kind of transaction.

I'd like to put $25/month into Crystal. I don't need a listing on Crystal's supporters page, but I would receive some sort of receipt (something that BountySource also appears to not be doing?), and I don't want to go through a platform I don't trust. What are my options?

_(FWIW, I'm supporting a number of open source projects though [OpenCollective](https://opencollective.com/), which has been a good experience so far. They even send receipts.)_

**Comment by Sija** [comment]
👍 for setting up OpenCollective account.

**Comment by arikast** [comment]
I also gave up on BountySource long ago, same frustrations.  Website's simply busted.

**Comment by blitline-dev** [comment]
Some issue here. We used to support via BountySource but it became to painful to use.

We support ImageMagick monthly through PayPal which is a much better setup.

**Comment by hmans** [comment]
A quick update regarding my own situation with BountySource: they've refunded those $30 (which I feel bad about because I had no problems with giving $30 to Crystal at all) and added $50 of credit to my account. https://twitter.com/Bountysource/status/991960275103334400

I appreciate that they acted on this in _some_ fashion, but it doesn't really fix the issue I'm having with them. I will try to forward those $50 to Crystal, but would still love to set up a regular support payment via a platform that is not BountySource.

**Comment by Sija** [comment]
🏓 @crystal-lang core team (psss! it's about your money, pay attention ;))

**Comment by bcardiff** [comment]
We've had some conversation with OpenCollective in the past far before this issue. There are some unresolved issues regarding how to get the money from OC to Manas, mainly because of regulatory boring things. Maybe we could give another round and see if it is workable.

Regarding where the money goes. It goes to Manas and they are spend only for crystal development hours. As it can be read in [a past blog post](https://crystal-lang.org/2017/12/19/this-is-not-a-new-years-resolution.html) currently at least twice the hours covered by donations are been allocated.

@wontruefree the link has been fixed

**Comment by wontruefree** [comment]
Thanks that explains a lot and is very open of Manas.

**Comment by felixbuenemann** [comment]
Btw. currently visiting https://www.bountysource.com/teams/crystal-lang/fundraisers/702-crystal-language (at least from Germany) results in the page starting to load and then it just hangs and eats up 100% CPU – tested in Safari, Chrome and FF. So I doubt it's a browser bug.

**Comment by hmans** [comment]
Happens here, too, @felixbuenemann.

**Comment by bcardiff** [comment]
@felixbuenemann I think that is the old link that changed on BS side. It was changed to https://salt.bountysource.com/teams/crystal-lang in some places, but if you still find it somewhere let us know where.

**Comment by felixbuenemann** [comment]
It is linked from the bountysource badge at the top of the README.md.

**Comment by nditada** [comment]
In the past we had struggled with Paypal as they had not find a legal pathway through the tax agency here (Argentina). 

I have checked that again and it looks like that has now been solved and we can open a business paypal account to receive payments via that channel.

We like OpenCollective a lot and will start the process of setting it up for Crystal. I'll update here when we have news.

Thanks!

**Comment by blitline-dev** [comment]
Any update on this? We'd like to donate money

**Comment by matiasgarciaisaia** [comment]
@blitline-dev thanks for your support! 💙

Haven't forgot about this issue, but we're talking Argentinian times here :)

Will let you know here as soon as we have an update.

**Comment by nditada** [comment]
Thanks for the patience! Here we are, any early feedback on the experience would be appreciated. And thanks a ton for the support 🙏 !  cc @blitline-dev @hmans 
https://opencollective.com/crystal-lang

**Comment by hmans** [comment]
Thanks for setting up an OpenCollective account.

To remind everyone that they should be avoided, BountySource today sent out this email:

![screen shot 2018-12-17 at 09 32 07](https://user-images.githubusercontent.com/1061/50075369-0f726200-01df-11e9-9952-221519ecbc2b.png)

Not only are they telling me this a full 16 days _after_ the start date of this fees, they also just deducted this fee from my account balance there right away. (This, fortunately, was "free" money they added to my account after the woes I had with them that triggered the creation of this issue, but if I had known about these inactivity fees _in advance_, I would have preferred to put it to better use than just letting them claim it back.)

tl;dr Avoid BountySource.

**Comment by nditada** [comment]
Hi Hendrik,

We had seen a notification of this a while ago and have been doing periodic withdrawals, but thanks for the heads up!

**Comment by jhass** [comment]
Anything left to do here?
